tidb-lightning/tidb-lightning-distributed-import.md

```diff
@@ -171,6 +171,11 @@ pattern = '(?i)^(?:[^/]*/)*my_db\.my_table\.(0[0-4][0-9][0-9][0-9]|05000)\.sql'
 schema = "my_db"
 table = "my_table"
 type = "sql"
+
+[tikv-importer]
+# Whether to allow importing data into tables that already have data. The default value is false.
+# When using parallel import, because multiple TiDB Lightning instances import a table at the same time, this configuration item must be set to true.
```

Comment:
L96、L97 的描述需要和这里保持一致。麻烦也修改下。
```suggestion
# Whether to allow importing data into tables that already have data. The default value is `false`.
# When using parallel import, because multiple TiDB Lightning instances import a table at the same time, this configuration item must be set to `true`.
```